doc/docs.md

```diff
@@ -310,6 +311,17 @@ fn main() {
 Unlike most languages, variable shadowing is not allowed. Declaring a variable with a name
 that is already used in a parent scope will cause a compilation error.
 
+An exception to this rule is shadowing imported module names:
+```v failcompile
+import time
+
+fn main() {
+    time := 'foo' // `time` module is now shadowed in this scope
```

Comment:
I feel like this example could be a little more representative of the cases when this feature is actually useful (`gg` for example)

Reply:
example now is very close to a real use

Reply:
much better, thanks